wazuh/wazuh-kibana-app
# Details flyout date picker

**Description:**

Changed the filter to take the details range and not the global range

**Issues Resolved:**

Vulnerabilities inventory details flyout date-picker [#3932](https://github.com/wazuh/wazuh-kibana-app/issues/3932)

**Test:**

1. Navigate to `Integrity monitoring > Inventory` or `Vulnerabilities > Inventory` 
2. Select any row from the table
3. Change date-picker values
4. See that the information in the table changes
5. Navigate to `Integrity monitoring > Events`
6. See that the global date-picker did not change

Screenshot:

![image](https://user-images.githubusercontent.com/63758389/161721919-16fb2939-cb9e-4c35-af61-6ec523959b6a.png)
![image](https://user-images.githubusercontent.com/63758389/161721960-14e5ed40-8d4e-47cb-8662-de2d3248a822.png)
![image](https://user-images.githubusercontent.com/63758389/161722004-6be379ae-56fb-46b0-b7ce-9152a03374bf.png)
